contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryLogicB2_3.sol

```diff
@@ -351,6 +353,10 @@ contract AutomationRegistryLogicB2_3 is AutomationRegistryBase2_3 {
     return i_allowedReadOnlyAddress;
   }
 
+  function getWrappedNativeTokenAddress() external view returns (address) {
+    return i_wrappedNativeTokenAddress;
+  }
```

Comment:
I think we could do `getWrappedNativeToken()` and return the actual `IWrappedNative`

Reply:
For consistency, I think address is better here. 

If you see the usage of these getters, they are mostly for the Base2_3 constructor, the the base constructor uses all addresses:

```
    AutomationRegistryBase2_3(
      logicA.getLinkAddress(),
      logicA.getLinkUSDFeedAddress(),
      logicA.getNativeUSDFeedAddress(),
      logicA.getFastGasFeedAddress(),
      logicA.getAutomationForwarderLogic(),
      logicA.getAllowedReadOnlyAddress(),
      logicA.getWrappedNativeTokenAddress()
    )
```

Reply:
ah I see. I think ideally we would have made all of those like `getLinkToken()` but I'm not hung up on it!